pkg/ctl/install/fluxinstaller_test.go

```diff
@@ -0,0 +1,100 @@
+package install
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
```

Comment:
All other packages use ginko/gomega, did you have a particular reason for not using it here?

Reply:
No reason. I will look into ginko gomega